src/app/workshops-list/workshops-list.component.ts

```diff
@@ -1,34 +1,79 @@
 import { Component, OnInit, Input } from '@angular/core';
-import { WorkshopRepository, IWorkshopOverview } from '../services/workshops/workshopRepository'
+import { WorkshopRepository, IWorkshopOverview, IWorkshopListDto, ILocation } from '../services/workshops/workshopRepository'
 import { Router } from '@angular/router';
 
 @Component({
     selector: 'workshops-list',
     templateUrl: './workshops-list.component.html',
-    styleUrls: ['./workshops-list.component.css']
+    styleUrls: ['./workshops-list.component.scss']
 })
 
 export class WorkshopsListComponent {
 
-@Input() path: string;
+    @Input() path: string;
+    workshopDto: IWorkshopListDto[];
     workshops: IWorkshopOverview[];
+    // private locationList: { [locationId: string]: ILocation; } = {};
+    private locationList = new Map<number, ILocation>();
+    private locations: ILocation[];
 
     constructor(private workshopRepository: WorkshopRepository, private router: Router) {
         this.workshops = [];
+        this.getLocationList();
```

Comment:
i have a new repository in services which stores  all of this static content. Will merge.